packages/twenty-docs/src/components/token-form.tsx

```diff
@@ -74,46 +105,61 @@ const TokenForm = ({
   }, []);
 
   return (
-    !isTokenValid && (
-      <div>
-        <div className="container">
-          <form className="form">
-            <label>
-              To load your playground schema,{' '}
-              <a
-                className="link"
-                href="https://app.twenty.com/settings/developers"
-              >
-                generate an API key
-              </a>{' '}
-              and paste it here:
-            </label>
-            <p>
-              <input
-                className={token && !isLoading ? 'input invalid' : 'input'}
-                type="text"
-                readOnly={isLoading}
-                placeholder="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIyMD..."
-                defaultValue={token}
-                onChange={updateToken}
-              />
-              <span
-                className={`token-invalid ${
-                  (!token || isLoading) && 'not-visible'
-                }`}
-              >
-                Token invalid
-              </span>
-              <div className="loader-container">
-                <TbLoader2
-                  className={`loader ${!isLoading && 'not-visible'}`}
-                />
-              </div>
-            </p>
-          </form>
+    <div className="form-container">
+      <form className="form">
+        <div className="backButton" onClick={() => history.push('/')}>
```

Comment:
Maybe history.goBack() would be best?

Reply:
I had this in place and it felt a bit weird especially with changing between the schemas using the dropdown - but its your call what's better ;)

Reply:
Ah yes good point I didn't think of that. What about using history.replace() instead of history.push() in the dropdown? Not sure if it's a good idea but that's the only simple option that comes to mind. If it doesn't work nevermind, I agree we should leave it to the home then.

Reply:
good Idea this seems to work better!